src/main/asciidoc/workingwithapachesyncope/adminconsole/reports.adoc

```diff
@@ -0,0 +1,26 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+===== Reports
+The reports tab presents the admin with the reports generated from various jobs run on the syncope
+deployment. These reports are displayed using report templates which can be defined for FO, HTML
+and CSV formats. This tab also allows the admin to create and edit such templates.
+
```

Comment:
Missing information:
 * reports can be run several times, each execution can be exported in the supported formats
 * reports can be defined by composing reportlet instances
 * predefined reportlets are available, more can be developed